library/core/src/lib.rs

```diff
@@ -20,11 +20,15 @@
 // FIXME: Fill me in with more detail when the interface settles
 //! This library is built on the assumption of a few existing symbols:
 //!
-//! * `memcpy`, `memcmp`, `memset`, `strlen` - These are core memory routines which are
-//!   often generated by LLVM. Additionally, this library can make explicit
-//!   calls to these functions. Their signatures are the same as found in C.
-//!   These functions are often provided by the system libc, but can also be
-//!   provided by the [compiler-builtins crate](https://crates.io/crates/compiler_builtins).
+//! * `memcpy`, `memmove`, `memcmp`, `memset`, `strlen` - These are core memory routines which are
```

Comment:
Does this need to include `bcmp` as well?  We never emit calls to it, but LLVM will turn `memcmp` calls into it if it's available:

https://github.com/rust-lang/rust/blob/474709a9a2a74a8bcf0055fadb335d0ca0d2d939/tests/codegen/array-equality.rs#L30

Reply:
I wondered about the bcmp in compiler-builtins. According to `man bcmp` it is deprecated...